frontend/front/src/components/SurveyComponent/SurveyComponent.js

```diff
@@ -208,7 +210,7 @@ const SurveyComponent = ({
                     key={`q${q.id}`}
                     control={control}
                     name={`${q.id}`}
-                    label={q.text}
+                    label={q.question}
```

Comment:
The field for the survey question "question" changed from text to question at some point...